<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Store</class>
 <widget class="QWidget" name="Store">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="base">
    <attribute name="title">
     <string>交付中心大小</string>
    </attribute>
    <widget class="QLabel" name="basesize">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="map">
    <attribute name="title">
     <string>地图大小</string>
    </attribute>
    <widget class="QLabel" name="mapsize">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mine">
    <attribute name="title">
     <string>矿脉个数</string>
    </attribute>
    <widget class="QLabel" name="mine0">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mine1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="weapon2">
    <attribute name="title">
     <string>回收金币</string>
    </attribute>
    <widget class="QLabel" name="money0">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="money1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="money2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="money3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="money4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>请选择属性</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="updatebt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitbt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="cost">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
